jobserver/models/outputs.py

```diff
@@ -154,3 +122,47 @@ def get_absolute_url(self):
     def get_api_url(self):
         """The API url that will serve up this file."""
         return reverse("api:release-file", kwargs={"file_id": self.id})
+
+
+class Snapshot(models.Model):
```

Comment:
Minor nit - I think I prefer `ReleaseSnapshot` more than plan snapshot, as we may have other kinds of snapshots, perhaps. But not a blocker.